### DISCUSSION TITLE
wrapLongitude of geojsonlayer

### DISCUSSION THREAD

**Discussion by yanshjie** [visgl/deck.gl#7275] [Q&A]
Does wrapLongitude of geojsonlayer work?

  The Doc says wrapLongtitude (https://deck.gl/docs/api-reference/core/layer#wraplongitude) will make the paths go the shorter side of the world. But it seems not work when I tried it on Codepen.

**Comment by Pessimistress** [comment]
> Does wrapLongitude of geojsonlayer work?

Yes.

> But it seems not work when I tried it on Codepen.

Don't know what you tried, don't know what to tell you.

**Reply by yanshjie** [reply]
Sorry to reply so late, I have been busy doing  other things.
But recently I have read the source code of PathLayer, and I notice that in Path.ts 
![图片](https://user-images.githubusercontent.com/98957105/193268891-71d5eb19-f163-4ff1-83a4-c1bd65c8ca4f.png)
The gridResolution property blocks the wrapLongitude operation and this property comes from viewport.resolution:
https://github.com/visgl/deck.gl/blob/1a42362a310b39b5b26d46354d388e2319a90af3/modules/layers/src/path-layer/path-layer.ts#L251
So the problem is, what is gridResolution doing, how do I set it to False?

By the way, I am actually using the EditableGeoJsonLayer from nebula.gl(https://nebula.gl/docs/api-reference/layers/editable-geojson-layer) and encounter the across 180 degree problem in GlobeView, but fortunately I have found a way to work around it.

Thank you to your reply!
